cpp/include/graph.hpp

```diff
@@ -522,117 +524,101 @@ class GraphCompressedSparseBase {
   {
-    return GraphCSRView<VT, ET, WT>(GraphCompressedSparseBase<VT, ET, WT>::offsets(),
-                                    GraphCompressedSparseBase<VT, ET, WT>::indices(),
-                                    GraphCompressedSparseBase<VT, ET, WT>::edge_data(),
-                                    GraphCompressedSparseBase<VT, ET, WT>::number_of_vertices(),
-                                    GraphCompressedSparseBase<VT, ET, WT>::number_of_edges());
+    return GraphCSRView<vertex_t, edge_t, weight_t>(
+      GraphCompressedSparseBase<vertex_t, edge_t, weight_t>::offsets(),
+      GraphCompressedSparseBase<vertex_t, edge_t, weight_t>::indices(),
+      GraphCompressedSparseBase<vertex_t, edge_t, weight_t>::edge_data(),
+      GraphCompressedSparseBase<vertex_t, edge_t, weight_t>::number_of_vertices(),
+      GraphCompressedSparseBase<vertex_t, edge_t, weight_t>::number_of_edges());
   }
 };
 
 /**
  * @brief       A constructed graph stored in CSC (Compressed Sparse Column)
  * format.
  *
- * @tparam VT   Type of vertex id
- * @tparam ET   Type of edge id
- * @tparam WT   Type of weight
+ * @tparam vertex_t   Type of vertex id
+ * @tparam edge_t   Type of edge id
+ * @tparam weight_t   Type of weight
  */
-template <typename VT, typename ET, typename WT>
-class GraphCSC : public GraphCompressedSparseBase<VT, ET, WT> {
+template <typename vertex_t, typename edge_t, typename weight_t>
+class GraphCSC : public GraphCompressedSparseBase<vertex_t, edge_t, weight_t> {
  public:
   /**
    * @brief      Default constructor
    */
-  GraphCSC() : GraphCompressedSparseBase<VT, ET, WT>() {}
+  GraphCSC() : GraphCompressedSparseBase<vertex_t, edge_t, weight_t>() {}
 
   /**
    * @brief      Take ownership of the provided graph arrays in CSR format
    *
-   * @param  offsets               This array of size V+1 (V is number of
-   * vertices) contains the
-   * offset of adjacency lists of every vertex. Offsets must be in the range [0,
-   * E] (number of
-   * edges).
-   * @param  indices               This array of size E contains the index of
-   * the destination for
-   * each edge. Indices must be in the range [0, V-1].
-   * @param  edge_data             This array of size E (number of edges)
-   * contains the weight for
-   * each edge.  This array can be null in which case the graph is considered
-   * unweighted.
    * @param  number_of_vertices    The number of vertices in the graph
    * @param  number_of_edges       The number of edges in the graph
+   * @param  has_data              Wiether or not the class has data, default = False
+   * @param  stream                Specify the cudaStream, default = null
+   * @param mr                     Specify the memory resource
    */
-  GraphCSC(VT number_of_vertices_,
-           ET number_of_edges_,
+  GraphCSC(vertex_t number_of_vertices_,
+           edge_t number_of_edges_,
            bool has_data_                      = false,
```

Comment:
variable names with trailing "_" is for class private member variables (RAFT clang-tidy rules).

Reply:
fixed